### ISSUE TITLE
Raise Error:  missing key 'python_full_version' 

### ISSUE THREAD

**Issue by hackrole** [oxidase/rules_poetry#58]
detail message below.
```  
File "/home/username/.cache/bazel/_bazel_username/0d8ea45d789d52c49f0973c6387b3cdd/external/rules_poetry~override/python/poetry_deps.bzl", line 122, column 57, in _package_impl
    deps = [dep for dep in ctx.attr.deps if _include_dep(dep, ctx.attr.markers, tags)]
File "/home/username/.cache/bazel/_bazel_username/0d8ea45d789d52c49f0973c6387b3cdd/external/rules_poetry~override/python/poetry_deps.bzl", line 66, column 26, in _include_dep
    return evaluate(parse(marker, environment))
File "/home/username/.cache/bazel/_bazel_username/0d8ea45d789d52c49f0973c6387b3cdd/external/rules_poetry~override/python/private/markers.bzl", line 124, column 21, in parse
    fail("missing key '{}' in environment {}".format(value, environment))

Error in fail: missing key 'python_full_version' in environment {"extra": "*", "implementation_name": "cpython", "platform_python_implementation": "CPython", "platform_tags": ["linux_x86_64", "manylinux2014_x86_64", "manylinux_2_12_x86_64", "manylinux_2_17_x86_64", "manylinux_2_27_x86_64", "manylinux_2_28_x86_64"], "python_version": "3.9", "os_name": "posix", "platform_machine": "x86_64", "platform_system": "Linux", "sys_platform": "linux"}

```

I check https://peps.python.org/pep-0496/#version-numbers, seems need to fix this rule to support it.

**Comment by oxidase** [comment]
With https://github.com/oxidase/rules_poetry/pull/59 it is possible to specify `python_full_version` explicitly like in ["markers" example](https://github.com/oxidase/rules_poetry/blob/6b147ce3/examples/markers/MODULE.bazel#L17-L38).

In case if Python toolchains are from `rules_python` I plan to add default values based on [mapping](https://github.com/bazelbuild/rules_python/blob/ced8cc6a/python/versions.bzl#L427-L433).

**Comment by oxidase** [comment]
@hackrole  thanks for reporting! Fixed in v0.3.5
